### PR TITLE
DEPL-9060 disable-pool-member.py fails to be executed when a overthere.Localhost is being used for a windows box

### PR DESCRIPTION
**Pull request by adityakalia**

### CODE REVIEW

**Comment by adityakalia** [comment]
@hierynomus please review

**Comment by hierynomus** [comment]
Can't we do something with the `CommandLine` class instead of the String[] manipulation?

**Comment by adityakalia** [comment]
I am assuming you are referring to com.xebialabs.overthere.CmdLine class, I did not make any changes their because it is used by other connection types as well, the changes are very specific to LocalConnection only.

**Comment by hierynomus** [comment]
Yes, but what I meant is that you could build a new `CmdLine` that starts with `'cmd', '/c'`. Instead of all the manual `String[]` manipulation.

**Comment by adityakalia** [comment]
ah missed it, please check now

File: src/main/java/com/xebialabs/overthere/local/LocalConnection.java [hunk 1]
```diff
@@ -129,7 +124,11 @@ public OverthereProcess startProcess(CmdLine cmd) {
 
         try {
             logger.debug("Creating " + os + " process with command line [{}]", obfuscatedCmd);
-            final ProcessBuilder pb = new ProcessBuilder(cmd.toCommandArray(os, false));
+            String[] args = cmd.toCommandArray(os, false);
```
Comment (hierynomus) [inline src/main/java/com/xebialabs/overthere/local/LocalConnection.java]: don't `toCommandArray` here, but check with the if, and pass the `CmdLine` as an argument to the method, and return the new `CmdLine` from it.

**Comment by adityakalia** [comment]
hope its fine now

**Comment by hierynomus** [comment]
Yay :+1: looks good now!

**Pull request merged**